lockfile/src/spdx.rs

```diff
@@ -74,14 +74,16 @@ pub(crate) struct Relationship {
 
 fn type_from_url(url: &str) -> anyhow::Result<PackageType> {
     if url.starts_with("https://registry.npmjs.org")
-        | url.starts_with("https://registry.yarnpkg.com")
+        || url.starts_with("https://registry.yarnpkg.com")
     {
         Ok(PackageType::Npm)
     } else if url.starts_with("https://rubygems.org") {
         Ok(PackageType::RubyGems)
     } else if url.starts_with("https://files.pythonhosted.org") {
         Ok(PackageType::PyPi)
-    } else if url.starts_with("https://repo1.maven.org") {
+    } else if url.starts_with("https://repo1.maven.org")
+        || url.starts_with("https://search.maven.org")
```

Comment:
This seems to go to `central.sonatype.com`. Is there a chance the name will change in the future and it should also be added immediately?

Reply:
I can add it, I think that change was fairly recent. The [API guide](https://central.sonatype.org/search/rest-api-guide/) still shows `search.maven.org`.